drivers/impuls-magnetic-mib.c

```diff
@@ -0,0 +1,281 @@
+/*  impuls_magnetic_mib.c - (ietf-mib.c + fix for upsload for IMPULSE SNMP-card for 1-10kVA UPS)  - data to monitor SNMP UPS (RFC 1628 compliant) with NUT
+ *
+ *  Copyright (C) 2002-2006
+ *  2012-2024   Sergei Kartavtsev <sergega@me.com>
+ *	2002-2012	Arnaud Quette <arnaud.quette@free.fr>
+ *	2002-2006	Niels Baggesen <niels@baggesen.net>
+ *	2002-2006	Arjen de Korte <adkorte-guest@alioth.debian.org>
+ *
+ *  Sponsored by MGE UPS SYSTEMS <http://www.mgeups.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ */
+
+#include "impuls-magnetic-mib.h"
+
+#define IMPULS_MAGNETIC_MIB_VERSION "1.0"
+
+/* SNMP OIDs set */
+#define IETF_OID_UPS_MIB "1.3.6.1.2.1.33.1."
+#define IETF_SYSOID ".1.3.6.1.2.1.33"
```

Comment:
Is there a specific SYSOID for Impulse MIB, so we could auto-detect that the SNMP server is this device (and so it could be listed at higher priority than `&ietf` in `snmp-ups.c` file, and not require "manual configuration" to be used).